docs/api/webview-tag.md

```diff
@@ -606,6 +606,23 @@ listening to the `channel` event with the [`ipcRenderer`](ipc-renderer.md) modul
 See [webContents.send](web-contents.md#contentssendchannel-args) for
 examples.
 
+### `<webview>.sendToFrame(frameId, channel, ...args)`
+
+* `frameId` Integer | [number, number] - the ID of the frame to send to, or a
+  pair of `[processId, frameId]` if the frame is in a different process to the
+  main frame.
```

Comment:
I'm a bit concerned about setting a precedence for accepting inflexible arguments here which would be difficult to change in the future. I'd feel much more comfortable if we could pass a `WebFrameMain` instance instead.

Part of my concern is that `FrameTreeNode` IDs might be a more stable singular identifier for frames which I might propose using in the future.

When #29290 is eventually merged, we could add a lazy `frame` property directly to the Event object. 
https://github.com/electron/electron/blob/44491b023ac2653538b7ac36bb73f3085a938666/shell/common/gin_helper/event_emitter.cc#L70-L74

Reply:
I also wonder whether adding such a convenience method is necessary when [`frame.send`](https://www.electronjs.org/docs/api/web-frame-main#framesendchannel-args) exists already.

With the lazy `frame` property, we could potentially have

```ts
webview.addEventListener('ipc-message', event => {
  event.frame.send(event.channel, ...event.args);
});
```

Reply:
With the understanding now that this API is added to the renderer, I think the API looks good overall.

Reply:
we cannot use `WebFrameMain` here as it's the renderer process